### PR TITLE
Bugfix, merging of dictionaries accidentialy overwrites illumina data

### PR DESCRIPTION
**Pull request by alneberg**
Affects the dashboard plot.

### CODE REVIEW

File: status/statistics.py [hunk 1]
```diff
@@ -242,14 +242,22 @@ def get(self):
...
+                    self.nanopore_flowcell_aggregates[fa][1],
+                    self.cleaning,
             )
+
+            # Use |= to merge the resulting dictionary with what's already
+            # inside data[fa][key], | works as a union for dictionaries.
+            # Doesn't work recursively though, so we have to do it for the bottom level only
+            for key, value in nanopore_stats.items():
+                if key in data[fa]:
+                    data[fa][key] |= value
```
Comment (remiolsen) [inline status/statistics.py:257]: I'm okay with this as long as the non-commutative(?) property of this union does not give problems. E.g given dicts `a` and `b`, `a | b != b | a`

Comment (alneberg) [reply status/statistics.py:257]: Yes, it's true that it is non-commutative. Colliding keys inside the two dictionaries `data[fa][key]` and `value` will take the _value_ of `value`... 😂 . The [PEP](https://peps.python.org/pep-0584/#specification) probably expresses this better:

``` If a key appears in both operands, the last-seen value (i.e. that from the right-hand operand) wins```

I the usecase I have in mind (sequencing throughput plot) I think this is perfectly fine, but of course that might not be the case for other use cases.

Since the code was written in a very general way before, I didn't want to dumb it down and write it specific instead... Do you think that would instead be good in this case?

**Review by remiolsen — APPROVED** [review]
This seems like a bit of niche python fuctionality, but I'm more than happy to approve it.
